playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -132,6 +132,7 @@ EDXAPP_CAS_ATTRIBUTE_PACKAGE: ""
 EDXAPP_ENABLE_AUTO_AUTH: false
 # Settings for enabling and configuring third party authorization
 EDXAPP_ENABLE_THIRD_PARTY_AUTH: false
+EDXAPP_ENABLE_OAUTH2_PROVIDER: false
```

Comment:
Sorry, my explanation wasn't clear.

This has never been included in the config.  Now that I'm adding it for the first time, my intention is that it's off for everybody but us.

When we want it, we can enable it specifically, by overriding it in our supplementary settings file.